go/libkb/full_self_cacher.go

```diff
@@ -166,7 +167,10 @@ func (m *CachedFullSelf) WithUser(arg LoadUserArg, f func(u *User) error) (err e
 	var u *User
 
 	if m.me != nil && m.isSelfLoad(arg) {
-		_ = m.maybeClearCache(ctx, &arg)
+		if arg.uid.IsNil() {
+			arg.uid = m.me.GetUID()
+		}
+		m.maybeClearCache(ctx, &arg)
```

Comment:
Use `arg.self` instead of `arg.uid.IsNil()` to determine this is a self load. So that bad args with neither continue to error out instead of getting self.

Reply:
Well... we're already checking for self above.  So how's this instead?

Reply:
My bad, extreme tunnel vision. Either way is good